### DISCUSSION TITLE
Carregando Inverno

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#383] [Imagens]
https://www.vish.blog.br/imagens/carregando-inverno/

**Comment by one-eyed-willy** [comment]
Se o inverno desse ano for pior que a virada de temperatura dos últimos dias vai ser tenso

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/carregando-inverno/#comment_1087421952>):

> Se o inverno desse ano for pior que a virada de temperatura dos últimos dias vai ser tenso

Não é? Eu acordei e vi nas notificações do celular e tava marcando 31° de temperatura. Fui dormir e vi que a do dia seguinte ia ser 10º mais frio

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/carregando-inverno/#comment_1087421952>):

> Se o inverno desse ano for pior que a virada de temperatura dos últimos dias vai ser tenso

Ainda assim, prefiro frio ao calor

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/carregando-inverno/#comment_1087425539>):

> Ainda assim, prefiro frio ao calor

Com certeza
